.github/workflows/env-checker-ci-pr.yml

```diff
@@ -49,12 +49,12 @@ jobs:
           fetch-depth: 0
 
       - name: Uninstalling .NET on Windows
-        if: ${{ matrix.os == 'windows-latest' }}
+        if: ${{ runner.os == 'Windows' }}
```

Comment:
I wonder why you don't use latest version after all the vscode test use latest version.

Reply:
These are version check statements. The original test state use "windows-latest" which depends on the version "latest". Now we have both macos-11 and macos-latest, use "runner.os == 'macOS'" can match both of them.